src/ISA/ISA.Json/Assay.fs

```diff
@@ -89,21 +89,3 @@ module Assay =
 
     //let toFile (path : string) (p:Assay) = 
     //    File.WriteAllText(path,toString p)
-
-
-module ArcAssay = 
```

Comment:
Now the default case for exporting an assay is the non isa-json conforming variant. Not sure about this.

Reply:
E.g. the guys from MIAPPE Wizard are already using ArcInvestigation.fromJsonString with isa-json conforming json as input. This would fail now.

Reply:
I would argue for keeping this function exactly as is and adding the new variants with some disambiguating suffix.